kernels/volk/volk_32f_s32f_convert_32i.h

```diff
@@ -431,14 +434,17 @@ static inline void volk_32f_s32f_convert_32i_a_generic(int32_t* outputVector,
     float min_val = INT_MIN;
     float max_val = INT_MAX;
     float r;
+    int s;
```

Comment:
The generic version doesn't benefit from an aligned version. We don't do anything fancy with aligned vectors besides choosing an implementation that expects aligned vectors.

I suggest to replace the function body with a call to the generic version. I'd support to remove this implementation entirely. However, we install all kernel headers and thus some users might use this function directly.